.github/workflows/coordinator-build-and-publish.yml

```diff
@@ -29,7 +29,7 @@ on:
 
 concurrency:
   group: coordinator-build-and-publish-${{ github.workflow }}-${{ github.ref }}
-  cancel-in-progress: ${{ github.ref_name != 'main' }}
+  cancel-in-progress: ${{ github.ref == 'refs/heads/main' }}
```

Comment:
What confuses me now is what if the ref branch is not main, then `cancel-in-progress` would be false now, whereas previous it would be true, is it expected? I lack the context here...

Reply:
You're correct, I've amended this. We want false on main and true when not on main.